src/web/views/enter-code.njk

```diff
@@ -34,5 +34,5 @@
 
     {# TODO DW HTBHF-1712 update link to send new confirmationCode  #}
     <p class="govuk-body-m"><a href="/send-code" id="request-new-code">{{ requestANewCode }}</a> {{ ifItDoesNotArrive }}</p>
```

Comment:
Is there a way of putting the link in the translation? This implementation assumes that all languages will use the same order of words.